group_project_v2/project_api/__init__.py

```diff
@@ -0,0 +1,28 @@
+''' API calls with respect group projects'''
+
+from django.conf import settings
+from lazy.lazy import lazy
+
+from group_project_v2.project_api.api_implementation import TypedProjectAPI
+
+# Looks like it's an issue, but technically it's not; this code runs in LMS, so 127.0.0.1 is always correct
+# location for API server, as it's basically executed in a neighbour thread/process/whatever.
+API_SERVER = "http://127.0.0.1:8000"
+if hasattr(settings, 'API_LOOPBACK_ADDRESS'):
+    API_SERVER = settings.API_LOOPBACK_ADDRESS
```

Comment:
Why not just the one-line version?
 `API_SERVER = getattr(settings, 'API_LOOPBACK_ADDRESS', 'http://127.0.0.1:8000')`